src/screens/signup/otpVerification/otpField.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import OTPInputView from '@twotalltotems/react-native-otp-input';
+import { useThemeContext } from '../../../theme';
+
+export default function OTPField() {
+  const { colors, fonts } = useThemeContext();
+  return (
+    <OTPInputView
+      pinCount={5}
+      style={{
+        width: 240,
```

Comment:
Use percentage here probably 60% given the page is 100% and you minus 20% from both sides as to fit all screen sizes